DEBIAN/control

```diff
@@ -0,0 +1,12 @@
+Version: 1.0.0
```

Comment:
do we need to update this on every release?

Reply:
Updating this as part of our dockerfile so updating here won't be necessary.